oci/layout/oci_dest.go

```diff
@@ -26,7 +26,7 @@ type ociImageDestination struct {
 // newImageDestination returns an ImageDestination for writing to an existing directory.
 func newImageDestination(ctx *types.SystemContext, ref ociReference) (types.ImageDestination, error) {
 	if ref.image == "" {
-		return nil, errors.Errorf("cannot save image with empty image.ref.name")
+		return nil, errors.Errorf("cannot save image with empty image.ref.name (must be of form <path>:<reference:tag>)")
```

Comment:
- This reads as if all of `image.ref.name` should be of form `<path>:<reference:tag>`, when `<path>:` is parsed into a different member.
- The `image.ref.name` annotation, per https://github.com/opencontainers/image-spec/blob/master/annotations.md#pre-defined-annotation-keys , does not have a “tag” concept at all.
- While you are cleaning this up, perhaps the error should say something easier to read/understand than `image.ref.name` (a _subset_ of the official annotation name).  `getManifestDescriptor` just calls it a “reference”, and the spec says “name of the reference” and “reference”.

Reply:
replaced image.ref.name with reference name and reworded what the syntax should look like. Should we clarify what we mean by reference name? the man page for buildah and podman does a good job at noting what the syntax should be like with examples also.

Reply:
> Should we clarify what we mean by reference name? 

If you can think of a concise way, sure.  It’s awkward that both the `image.ref.name` and the input to `ParseReference` are called a “reference”, and I can’t think of a nice way to differentiate in a memorable way (e.g. because they are really the same thing in the `docker://` transport, OTOH this error message may be from a caller which uses `oci/layout.ParseReference`, i.e. the user input may not contain a “transport” at all.)